app/models/score.rb

```diff
@@ -59,6 +60,10 @@ def log_entry
     " #{setter}")
   end
 
+  def log_delete
+    COURSE_LOGGER.log("Score #{id} DESTROYED for #{submission.course_user_datum.user.email}")
```

Comment:
I don't think we should log this, mainly because it can take a while for courses with more students, and it's not particularly useful.